core-web/libs/portlets/dot-ema/src/lib/dot-ema-shell/dot-ema-shell.component.ts

```diff
@@ -57,7 +48,18 @@ import { NavigationBarItem } from '../shared/models';
         }
     ],
     templateUrl: './dot-ema-shell.component.html',
-    styleUrls: ['./dot-ema-shell.component.scss']
+    styleUrls: ['./dot-ema-shell.component.scss'],
+    imports: [
```

Comment:
This component not need OnPush for any reason?

Reply:
This component just holds the navigation in edit EMA and since this component doesn't have `Input` nor `Output` Angular doesn't have a way to check this component for changes. Also the way angular works, if this components is not checked for changes nor the children will be checked, this made some portlets like Rules to not render until triggering some UI even like a double click and also that same portlet didn't work well because it also doesn't not emit nor takes any input